manager/manager.py

```diff
@@ -598,7 +598,7 @@ def generate_dashboard(self, max_failures=MAX_FAILURES):
                 '<th class="hdr">Repo</th>'
                 '<th class="hdr">PreConvert</th>'
                 '<th class="hdr">Converted</th>'
-                '<th class="hdr">Destination</th>'
+                '<th class="hdr">Destination</th>',
```

Comment:
Also, there should be a unit test that verifies you found the problem and your changes fix it.